### ISSUE TITLE
'NoneType' object has no attribute 'announce'

### ISSUE THREAD

**Issue by koakekuna** [drewsynan/velveeva-cli#1]
- What behavior were you expecting?
When executing: `velveeva go --go`, the program should nuke, bake, screenshot, package and clean.
- What actually happened?
The program exited with the error: `'NoneType' object has no attribute 'announce'`
- How can I reproduce it?
Happens on a few commands: --go, --clean, --dev, --veev2rel (but not --rel2veev), --watch
- What was your environment like?
All defaults when executing `velveeva init`. Nothing added into pre or post hooks. Not configured for FTP. No partials or templates.
- Firmware versions, host environment, hardware versions
MacOS High Sierra 10.13.6
Docker version 18.06.0-ce, build 0ffa825
cli version: 1.1.2
docker image sha: [drewsynan/velveeva@sha256:775deba2adf02456993091cf2cfc9effe458cec1aa46f1b78b7ccf982e71b20b]
- Do you have any logs?
none

**Comment by drewsynan** [comment]
Hey! Thanks for the issue report. I made a few changes that should fix the 'NoneType has no attribute announce' error.

A few caveats however:
- the `--watch` flag was completely non-functioning, and I've removed it. I think that it was a leftover from a refactor awhile back (?) I've found that using a separate watch/change dev server process is both speedier and a lot less painful.
- most of the `--go` like flags with multiple steps baked in don't work quite right. I've been meaning to get around to fixing the build planning/dependency graph stuff for while, but haven't gotten around to it. In the meantime, I've found that creating a script/makefile with a series of separate `velveeva go --(singlecommand)`s works best. e.g.:

```makefile
.PHONY : build
build :
	velveeva update
	velveeva go --nuke
	velveeva go --inline
	velveeva go --integrate
	velveeva go --rel2veev
	velveeva go --package

publish :
	velveeva go --controlsonly
	velveeva go --publishonly
```
Hope this helps!
